walletlibrary/src/test/java/com/microsoft/walletlibrary/requests/RequestHandlerFactoryTest.kt

```diff
@@ -1,74 +1,87 @@
 package com.microsoft.walletlibrary.requests
 
-import com.microsoft.walletlibrary.requests.handlers.RequestHandler
+import com.microsoft.walletlibrary.requests.handlers.RequestProcessor
 import com.microsoft.walletlibrary.requests.rawrequests.OpenIdRawRequest
+import com.microsoft.walletlibrary.requests.rawrequests.RawRequest
 import com.microsoft.walletlibrary.util.HandlerMissingException
 import com.microsoft.walletlibrary.util.UnSupportedRawRequestException
-import io.mockk.every
+import io.mockk.coEvery
 import io.mockk.mockk
+import kotlinx.coroutines.runBlocking
 import org.assertj.core.api.Assertions.assertThat
 import org.assertj.core.api.Assertions.assertThatThrownBy
 import org.junit.Test
 
 class RequestHandlerFactoryTest {
```

Comment:
Since RequestHandlers are RequestProcessors now, we could combine the tests in RequestProcessorTest class and remove this one.